source/frontend/src/features/mapSideBar/property/tabs/propertyDetailsManagement/detail/activitiesList/ManagementActivitiesListView.tsx

```diff
@@ -0,0 +1,67 @@
+import { orderBy } from 'lodash';
+import React from 'react';
+
+import LoadingBackdrop from '@/components/common/LoadingBackdrop';
+import { Section } from '@/components/common/Section/Section';
+import { TableSort } from '@/components/Table/TableSort';
+import { Api_PropertyManagementActivity } from '@/models/api/Property';
+
+import ManagementActivitiesList from './ManagementActivitiesList';
+import { PropertyActivityRow } from './models/PropertyActivityRow';
+
+export interface IManagementActivitiesListViewProps {
+  isLoading: boolean;
+  propertyActivities: PropertyActivityRow[];
+  onDelete: (managementActivityId: number) => void;
+}
+
+const ManagementActivitiesListView: React.FunctionComponent<IManagementActivitiesListViewProps> = ({
+  isLoading,
+  propertyActivities,
+  onDelete,
+}) => {
+  const [sort, setSort] = React.useState<TableSort<Api_PropertyManagementActivity>>({});
+
+  const mapSortField = (sortField: string) => {
```

Comment:
huh, was the display order not a viable option here?

Reply:
please expand.

Reply:
negative, display order is not a property on the activity. Such property was removed from the model.